File: example/javalib/script/3-custom-class/Qux.java
//| extends: [millbuild.LineCountJavaModule]
package qux;

public class Qux {
  public static String getLineCount() throws Exception {
    return new String(
        Qux.class.getClassLoader().getResourceAsStream("line-count.txt").readAllBytes());
  }

  public static void main(String[] args) throws Exception {
    System.out.println("Line Count: " + getLineCount());
  }
}

File: example/kotlinlib/script/4-custom-class/Qux.kt
//| extends: [millbuild.LineCountKotlinModule]
//| kotlinVersion: 2.0.20
package qux

import java.io.IOException

fun getLineCount(): String? = try {
    ::main.javaClass.classLoader
        .getResourceAsStream("line-count.txt")
        .readAllBytes()
        .toString(Charsets.UTF_8)
} catch (e: IOException) {
    null
}

fun main() {
    var msg = "Line Count: " + getLineCount()
    println(msg)
}
